xdg/desktop/neurodebian-fsl-5.0.desktop

```diff
@@ -2,9 +2,9 @@
 Encoding=UTF-8
 GenericName=Software Collection for (f)MRI Data Analysis
 GenericName[de]=Software-Kollektion zur (f)MRT-Daten Analyse
-Name=FSL 4.1
-Exec=nd-autoinstall -p fsl-4.1 fsl4.1-fsl
-Icon=fsl-4.1.xpm
+Name=FSL 5.0
+Exec=nd-autoinstall -p fsl-core
```

Comment:
since everything is versioned, lets for consistency make it fsl-5.0-core and be done.  I will merge and do it after.  THANKS!